app/bundles/NotificationBundle/Views/SubscribedEvents/Timeline/index.html.php

```diff
@@ -9,10 +9,11 @@
  * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
  */
 $data = $event['extra']['log']['metadata'];
-if (isset($data['failed'])) {
-    return;
-}
 ?>
+<?php if (isset($data['failed'])): ?>
+    <p class="text-danger mt-0 mb-10"><i class="fa fa-warning"></i> <?php echo $view['translator']->trans('mautic.campaign.event.last_error').': '.$data['reason']; ?></p>
+    <?php return; ?>
```

Comment:
Wow, haven't seen this in a HTML view yet. Does it mean you don't want to show the rest of the view? Can this be replaced with an `else` statement instead? I cannot find how to do this in a Twig template. And we'll need to rewrite this PHP template to TWIG pretty soon.